android/library/src/main/java/io/logto/android/LogtoConfig.kt

```diff
@@ -0,0 +1,20 @@
+package io.logto.android
+
+data class LogtoConfig(
+    val clientId: String,
+    val oidcEndpoint: String,
+    val scopes: List<String>,
+    val redirectUri: String,
+) {
+
+    fun getEncodedScopes(): String = scopes.joinToString(" ")
+
+    fun getAuthEndpoint(): String = "$oidcEndpoint$AUTH_PATH"
+
+    fun getTokenEndpoint(): String = "$oidcEndpoint$TOKEN_PATH"
```

Comment:
this should read from oidc discovery endpoint, please add a linear issue and ref it by comment here

Reply:
https://linear.app/silverhand/issue/LOG-76/retrieve-auth-config-from-oidc-discovery-endpoint